src/components/Dropdown/Dropdown.tsx

```diff
@@ -136,7 +142,7 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
 
       <ScreenReader
         announcementKey={screenReaderKey}
-        announcementText={isActive ? screenReaderText : ''}
+        announcementText={isActive && (readAutocomplete || items.length) ? screenReaderText : ''}
```

Comment:
What cases is this `readAutocomplete` boolean necessary? Trying to wrap my head around it but having a hard time

Reply:
So basically the screenreader isn't supposed to read the number of autocomplete options when the user first clicks into the searchbar, so readAutocomplete is set to true when the user starts typing and false when the user clicks out of the searchbar

Reply:
If there are autocomplete suggestions then it should still read it even if the user hasn't started typing, hence the `(readAutocomplete || items.length > 0)` logic

Reply:
The screenreader instructions are separate from the autocomplete announcement. We definitely want the instructions to read regardless of `readAutocomplete` when the user first starts interacting with the search bar. But, if there is some default input text before the user starts typing, the autocomplete announcement should also be made

Reply:
Not sure I understand, what would the other prop do?

Reply:
Sometimes, when users navigate to a page, we have automatically populated text present in the search bar even before they have started typing, which runs a default search on the page. So when they click on the search bar for the first time, there's already text there

Reply:
I think that should work!